pages/common/aria2c.md

```diff
@@ -12,7 +12,11 @@
 
 `aria2c -o {{filename}} {{url}}`
 
-- Download from multiple sources:
+- Download parallel from multiple sources each URI pointing to a different file:
+
+`aria2c -Z {{url_1}} {{url_2}}`
```

Comment:
We prefer long arguments if there are available
```suggestion
`aria2c --force-sequential {{url_1}} {{url_2}}`
```
Could you please expand the other examples?

Reply:
I have expanded all the short arguments I saw. Is there a way to specify both short and long arguments in tldr?

Reply:
Nop :disappointed: